src/ctia/http/middleware/auth.clj

```diff
@@ -32,6 +32,14 @@
 (defn wrap-authentication [handler]
   (testable-wrap-authentication handler @auth-service))
 
+(defn wrap-authenticated [handler]
+  (fn [request]
+    (let [auth-identity (:identity request)]
+      (if (and (not (nil? auth-identity))
+               (auth/authenticated? auth-identity))
+        (handler request)
+        (http-response/forbidden! {:message "Only authenticated users allowed"})))))
```

Comment:
The fact that this is not configurable make it impossible to spawn a fully public instance.
I think it's ok for now. But I believe someone might want to provide a fully open to the public instance even for non identified users.

Reply:
It's possible with the `allow-all` authentication mode. It always returns `true` to  `authenticated?`